interfaces/builtin/network_manager.go

```diff
@@ -0,0 +1,429 @@
+           send_interface="org.freedesktop.NetworkManager.Device.Team"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Device.Tun"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Device.Veth"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Device.Vlan"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.WiMax.Nsp"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.AccessPoint"/>
+
+    <!-- Devices (read-only, no security required) -->
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Device.WiMax"/>
+
+    <!-- Devices (read/write, secured with PolicyKit) -->
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Device.Wireless"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Device"/>
+
+    <!-- Core stuff (read-only properties, no methods) -->
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Connection.Active"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.DHCP4Config"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.DHCP6Config"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.IP4Config"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.IP6Config"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.VPN.Connection"/>
+
+    <!-- Core stuff (read/write, secured with PolicyKit) -->
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Settings"/>
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.Settings.Connection"/>
+
+    <!-- Agents; secured with PolicyKit.  Any process can talk to
+         the AgentManager API, but only NetworkManager can talk
+         to the agents themselves. -->
+    <allow send_destination="org.freedesktop.NetworkManager"
+           send_interface="org.freedesktop.NetworkManager.AgentManager"/>
+
+    <!-- Root-only functions -->
```

Comment:
I assume that we want to allow any tool running as root/service that has the network-manager plug to freely access those?